apps/web/pages/signin.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from 'ui';
+import { signIn } from 'next-auth/react';
+
+export default function SignIn() {
+  // added a separate function for this
+  // since we might need to specify different
+  // callback urls in the future
+  function handleSignIn() {
+    signIn('google');
+  }
+
+  return (
+    <div className='grid place-items-center h-screen'>
+      <section className='max-w-lg border-2 rounded-lg p-10 shadow-xl'>
+        <h1 className='mb-5 text-center text-3xl font-extrabold'>
+          Get more opportunities
+        </h1>
+        <h2 className='mb-7 text-center text-xl font-bold'>
+          And level up your career, with us
+        </h2>
+        <div className='flex justify-center'>
+          <Button
+            variant='ghost'
+            className='flex px-6 py-2 font-semibold select-none border-2 rounded-md text-white bg-slate-800'
+            onClick={handleSignIn}
+          >
+            <svg
```

Comment:
Sorry not related to this PR but could you set up [this]
(https://www.npmjs.com/package/@svgr/webpack)
You will need to add some changes to the next.config.js file
something like [this](https://github.com/project-shelf/project-shelf/blob/a4146a09fa9d9b2e813955df3516914cb6dce8de/apps/web/next.config.js#L11)
Once that's done you can  move this to its own .svg file, I think you can put in `public/assets/svgs/file.svg` and import it as a component.

Reply:
true we should do that, I will do that for the spinner in our button as well

Reply:
thanks, it might be a bit more tricky there because of the props but some magic with clsx should be enough. The other thing is that we are using esbuild on the `ui` library, so you will need to figure out how to load the svg there. If it's too complicated you can leave it.
This looks like it could do the job https://www.npmjs.com/package/esbuild-plugin-svgr